diff --git a/src/serialport/qserialport_unix.cpp b/src/serialport/qserialport_unix.cpp
index 342783c..ae94764 100644
--- a/src/serialport/qserialport_unix.cpp
+++ b/src/serialport/qserialport_unix.cpp
@@ -108,14 +108,14 @@ QT_BEGIN_NAMESPACE
 QString serialPortLockFilePath(const QString &portName)
 {
     static const QStringList lockDirectoryPaths = QStringList()
-        << QStringLiteral("/var/lock")
-        << QStringLiteral("/etc/locks")
-        << QStringLiteral("/var/spool/locks")
-        << QStringLiteral("/var/spool/uucp")
+    //    << QStringLiteral("/var/lock")
+    //    << QStringLiteral("/etc/locks")
+    //    << QStringLiteral("/var/spool/locks")
+    //    << QStringLiteral("/var/spool/uucp")
         << QStringLiteral("/tmp")
-        << QStringLiteral("/var/tmp")
-        << QStringLiteral("/var/lock/lockdev")
-        << QStringLiteral("/run/lock")
+    //    << QStringLiteral("/var/tmp")
+    //    << QStringLiteral("/var/lock/lockdev")
+    //    << QStringLiteral("/run/lock")
 #ifdef Q_OS_ANDROID
         << QStringLiteral("/data/local/tmp")
 #elif defined(Q_OS_OSX)
